roles/ceph-common/defaults/main.yml

```diff
@@ -282,9 +281,11 @@ mds_max_mds: 3
 ## Rados Gateway options
 #
 #radosgw_dns_name: your.subdomain.tld # subdomains used by radosgw. See http://ceph.com/docs/master/radosgw/config/#enabling-subdomain-s3-calls
+radosgw_resolve_cname: false # enable for radosgw to resolve DNS CNAME based bucket names
 radosgw_civetweb_port: 8080 # on Infernalis we get: "set_ports_option: cannot bind to 80: 13 (Permission denied)"
 radosgw_civetweb_bind_ip: "{{ ansible_default_ipv4.address }}"
 radosgw_civetweb_num_threads: 50
+radosgw_frontends: "civetweb port={{ radosgw_civetweb_bind_ip }}:{{ radosgw_civetweb_port }} num_threads={{ radosgw_civetweb_num_threads }}"
```

Comment:
Sure, maybe it best thing would be to add a comment with a link to https://github.com/civetweb/civetweb/blob/master/docs/UserManual.md? SSL, logging, and timeouts are probably the top options we're interested in but there are a ton of others.